src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBanner.vue

```diff
@@ -37,6 +40,7 @@ export default {
 			} else if (this.promoEnabled) {
```

Comment:
Since these are all returning something, I think in this case this just needs to be an `if` on it's own line, and the linting should pass

Reply:
resolved